custom/ilsgateway/tanzania/handlers/la.py

```diff
@@ -23,6 +23,10 @@ def _create_stock_transaction(self, report, product_id, quantity):
         )
 
     def handle(self):
+        sql_location = self.user.sql_location
+        if not sql_location or sql_location.location_type.administrative:
```

Comment:
I've just realized #11095 it's not enough because they might be some other handlers that can't handle non facilities users.